src/native/s3_client.c

```diff
@@ -1039,6 +1040,13 @@ JNIEXPORT jlong JNICALL Java_software_amazon_awssdk_crt_s3_S3Client_s3ClientMake
         checksum_config.validate_checksum_algorithms = &response_checksum_list;
     }
 
+    uint64_t object_size_hint = 0;
+    if (jni_object_size_hint) {
+        if (aws_uint64_t_from_java_Long(env, &object_size_hint, jni_object_size_hint, "object_size_hint")) {
```

Comment:
Java users never see this as `"object_size_hint"`. Maybe "object size hint" or "S3MetaRequestOptions.withObjectSizeHint"?

Reply:
Thanks, updated it to objectSizeHint.